core/src/main/java/nl/nn/adapterframework/filesystem/ExchangeFileSystem.java

```diff
@@ -890,10 +894,11 @@ public void createFolder(String folderName) throws FileSystemException {
 
 	@Override
 	public void removeFolder(String folderName, boolean removeNonEmptyFolder) throws FileSystemException {
-		ExchangeService exchangeService = getConnection();
+		ExchangeFolderReference reference = asFolderReference(folderName);
+		ExchangeService exchangeService = getConnection(reference);
 		boolean invalidateConnectionOnRelease = false;
 		try {
-			FolderId folderId = getFolderIdByFolderName(exchangeService, folderName, false);
+			FolderId folderId = getFolderIdByFolderName(exchangeService, folderName, reference, false);
```

Comment:
In deze call zou geen folderName als argument moeten zitten, want die zit al in reference